src/bin/am/commands/start.rs

```diff
@@ -96,6 +96,14 @@ pub struct CliArguments {
     )]
     pushgateway_version: String,
 
+    #[clap(
+        long,
+        env,
+        default_value = "https://explorer.autometrics.dev/static",
```

Comment:
Won't this value result in `/static/static`? 

The original request would hit `localhost:6789/explorer/static`, that path would be replaced with `/static` (see https://github.com/autometrics-dev/am/pull/141/files#diff-1bb7929eeafa7f90554634d7d3ae6c080a4352ac98c8a06f12691a5a51544aa9R13) and then that path (`/static`) would be appended to this base path.

Reply:
I guess this works since the path that is being joined is an absolute path, so it would remove any path from the original path. Guess that is fine since it is an mostly internal flag anyway.

Reply:
I've removed the '/static' from the default_value